packaging/postinstall.sh

```diff
@@ -73,6 +73,7 @@ chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME $IRODS_HOME/msiExecC
 chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME $IRODS_HOME/msiExecCmd_bin/univMSSInterface.sh.template
 chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME $IRODS_HOME/msiExecCmd_bin/irodsServerMonPerf.template
 chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME $IRODS_HOME/msiExecCmd_bin/hello
+chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME /var/run/irods
```

Comment:
Wondering if `/run/irods` needs to be listed here as well.

Tests are passing, so it's not yet clear.

@SwooshyCueb Thoughts?

Reply:
if it's just ours, then it seems correct to chown it too

Reply:
It does seem that way. However, `/run` is the modern location for runtime data. `/var/run` is a legacy path which is allowed to be a symlink to `/run`. FHS 3.0 states that applications using `/var/run` are allowed to continue to use it for the purposes of backwards compatibility.

This PR uses `/var/run` because CMake's GNUInstallDirs produces that prefix. While L76 doesn't rely on CMake, there's nothing saying it couldn't.

Based on what I've read, seems L76 should either be generated from the cmake or we add some logic that checks for `/run` and `/var/run` and proceeds accordingly.

Refs:
- https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html
- https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s13.html
- https://cmake.org/cmake/help/v3.21/module/GNUInstallDirs.html

Reply:
FreeBSD doesn't appear to have `/run`. They do have `/var/run` though.
- https://man.freebsd.org/cgi/man.cgi?hier

Reply:
Yes, I think this should be generated from CMake, as it is possible for the packager to override the values produced by GNUInstallDirs